test/pipeK.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+var S = require('..');
+
+var eq = require('./internal/eq');
+
+
+test('pipeK', function() {
+
+  eq(typeof S.pipeK, 'function');
+  eq(S.pipeK.length, 2);
+  eq(S.pipeK.toString(), 'pipeK :: Chain m => Array Function -> m a -> m b');
+
+  var xs = S.Just([1, 2, 3]);
```

Comment:
I would prefer to avoid this assignment. I like assertions to require as little context as possible.